README.md

```diff
@@ -1,37 +1,77 @@
-# Theatria
+# Theatria Wiki
```

Comment:
This main readme is distinct from the wiki. It's oriented around describing Theatria as a whole so the "wiki" portion doesn't need to be here. It's primarily to describe the project on github.

I would just leave this part of the repo the way it was for now.